### DISCUSSION TITLE
wLUNA lost when transferred from Ethereum to Solana via Wormhall

### DISCUSSION THREAD

**Discussion by albertjpn** [wormhole-foundation/wormhole#279] [Problems / bugs] [answered]
After the successful two transfers of wLUNA from Ethereum to Solana via Wormhole, I tried the third transfer of wLUNA from Ethereum to Solana. However, although the transaction at Ethereum succeeded and my wLUNA disappeared from my Metamask, wLUNA has not arrived at Phantom Wallet.

Eth tx: https://etherscan.io/tx/0xb62301b4042b7a00f908d9b1021c4665e708840dfd6e4c24e3671510275fa492
My eth wallet: 0x312C3F85a5b73B63583Cf1b7b45b5646A2812C4A
My solana wallet: 6iSLNqHhP3yAJ38Z4g5c6rWpjadyvfGMp326A5QD7qvz

The value of the affected asset is currently worth 63,000USD, which is most of my entire assets of my life and I cannot live without it.

Please, please help me as a matter of urgency.

**Comment by albertjpn** [comment]
Happy to provide screenshots or info you may need. Thank you.

**Comment by hendrikhofstadt** [comment, marked as answer]
Completed the transfer in https://explorer.solana.com/tx/254MfYQzBUnUiUmxSMWkHtJFyyXnA3tBm1EYjmotAezAYWbCqXgGt11EYH7AN31DgZPTCCg3n8ZEwK9LJsyVcETR

**Reply by albertjpn** [reply]
Thank you. I love your support and, more importantly, providing us with such an extremely helpful bridge.
